src/pk-transaction.c

```diff
@@ -2484,7 +2484,7 @@ pk_transaction_strvalidate (const gchar *text, GError **error)
 				     "Invalid input passed to daemon: zero length string");
 		return FALSE;
 	}
-	if (length > 1024) {
+	if (length == 1024) {
```

Comment:
Maybe >= might be safer here? Thanks for the PR!

Reply:
All other function calls to pk_strlen() use == but feel free to change the commit ;)

Reply:
@hughsie are you satisfied with the uniformity of `==` or do you _want_ me to change it to `>=` ?

Reply:
He'd prefer you to change it, as would I.

Reply:
Did that (sorry for the lag). Let me know if you wish me to go on patching all of the other checks for pk_strlen's output.

Reply:
Please do so, thanks! And squash the commit once you're done.